helpers/announce.js

```diff
@@ -17,6 +17,7 @@ export function announce(message) {
 		container.setAttribute('aria-live', 'polite');
 		container.style.display = 'inline-block';
 		container.style.position = 'fixed';
+		container.style.height = '0px';
```

Comment:
Unit should be left off when the value is zero.

```suggestion
		container.style.height = '0';
```

Reply:
(Side note... I thought the linter checked for this? 🤔 )

Reply:
The CSS linter does, but technically this is JavaScript writing CSS and it's not that smart.

Reply:
Ah, makes sense